.changeset/tiny-days-dance.md

```diff
@@ -0,0 +1,26 @@
+---
+'astro': minor
+---
+
+Cookie encoding / decoding can now be customized
```

Comment:
What's the convention for ending periods @sarah11918 ? Curious for future changesets and consistency

Reply:
The first line gets used as a heading in the blog post, so that's why I always omit the period there.

Reply:
Gotcha, thanks!